<launch>
    <!-- Load joint controller configurations from YAML file to parameter server -->
    <rosparam file="$(find march_gait_scheduler)/config/config_test_joint.yaml" command="load"/>

    <!--Setting the planning group to be used by the MoveitController, the choice being between full_exo and test_joint-->
    <param name="/control_node/planning_group" value="test_joint" />

    <!-- URDF description of the exoskeleton -->
    <include file="$(find march_description)/launch/load_test_joint_description.launch"/>

    <!-- By default, we do not start a database (it can be large) -->
    <arg name="db" default="false"/>

    <!-- By default, we are not in debug mode -->
    <arg name="debug" default="true"/>

    <!-- Load the joint controllers -->
    <node name="controller_spawner" pkg="controller_manager" type="spawner" ns="/march" output="screen"
          args="trajectory_controller joint_state_controller"/>

    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
          output="screen" ns="/march">
    </node>

    <node name="gait_scheduler_node" pkg="march_gait_scheduler" type="gait_scheduler_node" output="screen"/>

    <!-- OPTIONAL: Launch the rqt plugins that allow for easy debugging.-->
    <!-- Run Rviz and load the default config to see the state of the move_group node -->
    <arg name="rviz" default="true" doc="Launch rviz"/>

</launch>
